src/main/webapp/app/exercises/file-upload/participate/file-upload-submission.service.ts

```diff
@@ -69,7 +70,7 @@ export class FileUploadSubmissionService {
         if (lock) {
             url += '?lock=true';
         }
-        return this.http.get<FileUploadSubmission>(url);
+        return this.http.get<FileUploadSubmission>(url).pipe(map((res: ModelingSubmission) => this.convertItemFromServer(res)));
```

Comment:
you have a type here, this should be FileUploadSubmission

Reply:
Done ✅